src/scripts/wildBattle/RouteHelper.ts

```diff
@@ -38,6 +38,38 @@ class RouteHelper {
         return pokemonList;
     }
 
+    public static routePokerusEVs(route:number, region:GameConstants.Region): string {
+        const possiblePokemon: PokemonNameType[] = RouteHelper.getAvailablePokemonList(route, region);
+        if (this.minPokerus(possiblePokemon) == 3) {
```

Comment:
Use Gameconstant to make the code more readable

Reply:
Done :)